### ISSUE TITLE
Layers not visible: gray too light

### ISSUE THREAD

**Issue by ibarzrm** [sitmun/sitmun-viewer-app#92]
**Is your feature request related to a problem? Please describe.**
Layers not visible in the viewer's list of loaded layers. They are a very light gray. This may be too light, according to accessibility guidelines.

**Describe the solution you'd like**
Accessibility guidelines indicate that the contrast between text and background should be high. Therefore, the current grayscale may be too low and should be increased slightly.

**Comment by fjlopez** [comment]
Check if the styles are conform to accessibility rules

**Comment by fjlopez** [comment]
@ibarzrm I think that this is fixed in the new version of the viewer; add a screenshot if it is not yet fixed.

**Comment by ibarzrm** [comment]
We reopened the case because we realised that now, when the layers are not visible, they do not change colour.

In the following image, only ‘Municipalities at a scale of 50,000’ are visible, while in Loaded Layers, all appear in maroon.

<img width="1253" height="451" alt="Image" src="https://github.com/user-attachments/assets/521d6990-a3cd-4719-b071-6aa53ca9f843" />


**Possible solution:**
Perhaps the same grey (#777777) as the group title or a darker one could be used for the layer title.

**Comment by fjlopez** [comment]
**May update**: Contrast tweaks are partly in place in viewer CSS; the reopened case (off-map layers not visually distinct in “Loaded layers”) is still open pending a reproduce + DOM check against current SITNA.

**Comment by fjlopez** [comment]
Validated DONE: Capas/catalog not-visible rows use #777777 (custom-specific.css); CHANGELOG #92.
